### DISCUSSION TITLE
Installation on Nextcloud Pi 22.2.2

### DISCUSSION THREAD

**Discussion by stony2k** [thrillfall/nextcloud-gpodder#48] [Q&A]
Hi,

I installed Nextcloud Pi 22.2.2 and can't finde your app in the multimedia category. How can I install this app? Please help me :)

**Comment by JonOfUs** [comment]
I can find it on my Pi with native Nextcloud 22.2.3.
The app needs at least php7.4, could that be the problem?

If you still have php7.3 you could check whether upgrading to Raspbian Bullseye is working by now and upgrade your system to it (Bullseye includes php7.4) or you could use sury.org as an external package source (potential security problem). This way, you could even install the new php8.0 which has increased performance.
Tutorials on how to set up sury.org as package source can be found online (just search for 'raspberry pi install php7.4 sury' or something). I personally use sury as 2nd package source, but, as said before, it can be argued that this can be a potential security threat.

**Comment by stony2k** [comment]
My first attemend to install Nextcloud Pi was with the Raspbian Bullseye image. I installed it on an Raspberry Pi 2 and the Nextcloud Pi told me that the distribution is not campatbile. I flashed the legacy 32 bit version and managed to the Nextcloud Pi working.

The gPodder Sync app is listed on the official Nextcloud page: https://apps.nextcloud.com/apps/gpoddersync

Is php 7.4 or php 8 and sury.org as an external package source nessaray for seeing and installation the app in the Nextcloud Dashboard app section?

**Reply by JonOfUs** [reply]
It is necessary to have at least php7.4 in order to see the app in your Nextcloud's app manager. How you get php7.4 is not relevant, so you don't need sury.org if you have another source for it.

Hmm, yeah, seems to be an [open issue](https://github.com/nextcloud/nextcloudpi/issues/1321) with NextcloudPi and Bullseye support.

It seems like your only two options are to wait for Bullseye support in NextcloudPi or to install a newer version of php using other package sources (don't forget to configure your environment to use the new php version). i don't want to recommend non-official package sources without reservation, so i would definitely inform myself about the risks, but i never had any problems with sury.org and it is also used by many.
